integration/go_integration.md

```diff
@@ -0,0 +1,38 @@
+# Integration with external tools
+
+## Integration with bug tracking and story management tools
+
+Go allows you to link your commit messages with bug tracking and story management tools which are web applications.
+
+The following shows an example of tracking cards on Mingle by card numbers starting with a "\#" (e.g: \#4618):
```

Comment:
[This PR](https://github.com/gocd/gocd/pull/3021) will probably invalidate this section soon. Do you want to wait? Ganesh asking for a rebase of that PR. So, it might get merged soon.